src/main/java/dynamok/source/TaskConfig.java

```diff
@@ -45,6 +50,23 @@
         region = Regions.fromName(getString(Keys.REGION));
         topicFormat = getString(Keys.TOPIC_FORMAT);
         shards = Arrays.stream(getString(Keys.SHARDS).split(",")).filter(shardId -> !shardId.isEmpty()).collect(Collectors.toList());
+
+        String access, secret;
+        try {
+            access = getString(Keys.ACCESS_KEY_ID);
+        }
+        catch (ConfigException e) {
+            access = "";
+        }
+        accessKeyId = access;
```

Comment:
I would replace this with `accessKeyId = props.getOrDefault(Keys.ACCESS_KEY_ID, "")` and not use `getString()` at all.